.github/workflows/manual-nixos.yml

```diff
@@ -1,9 +1,8 @@
 name: "Build NixOS manual"
 
-permissions: read-all
-
 on:
-  pull_request_target:
+  push:
```

Comment:
Maybe we only want to run this on master branch than.

Reply:
Why not also for releases/staging?

Reply:
Release branch doesn't really see that many changes to it's docs/options after branch off. IMO hydra is sufficient for stable, it's unstable where the docs breakage was reasonably common.

PRs targeting staging would likely timeout frequently with mass rebuilds. Any errors missed in the initial PR on staging are caught by the action in the staging-next -> master PR.

Reply:
Yes. Mass-rebuilds is something you want to avoid in github actions. Especially without caching.